nemo/collections/asr/data/audio_to_text.py

```diff
@@ -1006,17 +1006,17 @@ def __init__(
         world_size: int = 0,
         return_sample_id: bool = False,
     ):
-        if use_start_end_token and hasattr(tokenizer, 'bos_token'):
```

Comment:
Why are these changes necessary? It used to work before with the values set here.